geomesa-plugin/src/main/scala/org/locationtech/geomesa/plugin/wcs/GeoMesaCoverageReader.scala

```diff
@@ -63,7 +63,7 @@ class GeoMesaCoverageReader(val url: String, hints: Hints) extends AbstractGridC
     true
   }
 
-  override def getOriginalEnvelope = this.getOriginalEnvelope
+  override def getOriginalEnvelope = getBounds
```

Comment:
actually, it would probably be better to just to set it to "this.originalEnvelope", as the call to getBounds is occurring at init anyways.